templates/feature/README.md

```diff
@@ -112,6 +116,14 @@ Projects created with `data-visuals-create` support two of the built-in ways tha
 
 The interface with Google Drive within `data-visuals-create` projects currently only supports using Oauth2 credentials to speak to the Google APIs. This requires a set of OAuth2 credentials that will be used to generate and save an access token to your computer. `data-visuals-create` projects have hardcoded locations for the credential file and token file, but you may override those with environmental variables.
 
+### Chrome
```

Comment:
Sounds good to me! It'd technically work for features if they contain the `.graphic` class somewhere, but you're right. I think this just adds extra noise since we'll probably mostly use them in actual graphics only projects. Removed in https://github.com/texastribune/data-visuals-create/pull/68/commits/63c07debf3cfc3250d1ac13046633e72ec8417a0 ✅